Sources/PackageModel/Manifest/TargetDescription.swift

```diff
@@ -227,3 +221,19 @@ extension TargetDescription.Dependency: ExpressibleByStringLiteral {
         self = .byName(name: value, condition: nil)
     }
 }
+
+import protocol Foundation.LocalizedError
```

Comment:
needed?

Reply:
We'll lose any textual content in certain cases if we don't provide a `LocalizedError`. One case is apparently the wrapping of errors that happens until we get to the test.